lib/runner.js

```diff
@@ -147,7 +152,8 @@ exports.execute = function (scripts, options, reporter, callback) {
         },
         reporter: reporter,
         filters: filters,
-        options: settings
+        options: settings,
+        only: onlyNodes[0]
```

Comment:
I think that'll require a comment for further maintenance, it's not obvious why you picked the 1st.